src/connectors/index.ts

```diff
@@ -39,3 +40,11 @@ let networkLibrary: providers.Web3Provider | undefined
 export function getNetworkLibrary(): providers.Web3Provider {
   return (networkLibrary = networkLibrary ?? getLibrary(network.provider))
 }
+
+// walletlink implements Metamask's RPC and should respond to most it's methods: window.ethereum.isMetaMask === true
+// More info: https://github.com/walletlink/walletlink
+export const walletlink = new WalletLinkConnector({
+  url: `https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`,
+  appName: "swapr",
```

Comment:
App name should be title case, we should add Swapr's logo as well
```suggestion
  appName: 'Swapr',
  appLogoUrl: SWAPR_LOGO
```